.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+## Description
+
+[Describe what changes you have made and why you have made them]
+
+## Related Issue
+
+[Add the link to the issue you have addressed or created, if applicable]
+
+## Checklist
+
+Please ensure that your pull request meets the following requirements:
+
+[ ] I have tested the changes locally and verified that they work as intended.
```

Comment:
```suggestion
- [ ] I have tested the changes locally and verified that they work as intended.
```

Reply:
add that dashes (`-`) at the beginning of these lines. reference -> https://www.markdownguide.org/extended-syntax/, look for a title called **Task Lists**